### PR TITLE
Remove assumption of cs50/ prefix

### PR DESCRIPTION
**Pull request by brianyu28**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,5 +61,5 @@ class CustomInstall(install):
         "console_scripts": ["submit50=submit50:main"]
     },
     url="https://github.com/cs50/submit50",
-    version="2.4.1"
+    version="2.4.0"
```
Comment (dmalan) [inline setup.py]: Just to be sure, is this downgrade correct?

File: submit50.py [hunk 2]
```diff
@@ -413,11 +413,6 @@ def submit(org, problem):
         raise Error(_("You have an old version of submit50. "
                       "Run update50, then re-run {}!".format(org)))
 
-    # assume cs50/ problem if problem name begins with a year
-    branch = problem
-    if problem.split("/")[0].isdigit():
-        branch = os.path.join("cs50", problem)
-
     # ensure problem exists
```
Comment (dmalan) [inline submit50.py]: "branch"?

File: submit50.py [hunk 3]
```diff
@@ -413,11 +413,6 @@ def submit(org, problem):
...
     file, submit.EXCLUDE = tempfile.mkstemp()
     url = "https://cs50.me/excludes/{}/".format(branch)
```
Comment (dmalan) [inline submit50.py]: This is changing now too, right?

And what if branch has `@` in it now?

File: submit50.py [hunk 4]
```diff
@@ -568,8 +563,7 @@ def submit(org, problem):
 
     # successful submission
     if org == "submit50":
-        cprint(_("Submitted {}! "
-                 "See https://cs50.me/submissions/{}.").format(problem, branch),
+        cprint(_("Submitted {}! See https://cs50.me/submissions.").format(branch),
```
Comment (dmalan) [inline submit50.py]: Mind updating PO file too? Manually should suffice.

File: setup.py (hunk 1 above)
Comment (brianyu28) [inline setup.py]: Yeah, don't know why it was 2.4.1 before, we want to bump fro 2.3.1 to 2.4.0 in the next release.

File: submit50.py (hunk 2 above)
Comment (brianyu28) [inline submit50.py]: Not really checking if the branch exists, right now just checking if there's an exclude file for the problem, which would indicate that the problem exists.

File: submit50.py (hunk 3 above)
Comment (brianyu28) [inline submit50.py]: Yep, see the other PR!

File: submit50.py (hunk 4 above)
Comment (brianyu28) [inline submit50.py]: Updated!

**Pull request merged**